### ISSUE TITLE
Incorrect animation cursor with repeating signs

### ISSUE THREAD

**Issue by josyjanga** [paulrosen/abcjs#171]
The animating cursor seems to recognize repeating signs incorrectly. If you try the following notation with the configurator, you will see that the cursor starts from the first line again, when it reaches the end of the first repeating part.
(I also tried animating the beats when playing the audio, which works just fine. Only the animation cursor seems to follow another logic.)
~~~~
X:1
L:1/8
Q:1/4=88
M:2/4
V:1 treble 
%%staffsep 90pt
V:1
K:G
"G" [Bd][GB] "D7" [Ac][DA] |"G" ([B,G]/[GB]/)[B,D] [B,G][GB] |
"D7" ([DF]/[FA]/)[CD] [DF][CA] |"G" .[B,G].[B,G].[B,G] z | !
|: "_I""G" [Bd][GB] "D7" [Ac][DA] | "G" ([B,G]/[GB]/)[B,D] [B,G][GB] |
"D7" ([DF]/[FA]/)[CD] [DF][CA] | "G" ([B,G]/[GB]/)[B,D] [B,G][GB] |
"G" [Bd][GB] "D7" [Ac][DA] |"G" ([B,G]/[GB]/)[B,D] [B,G][GB] | 
"D7" ([DF]/[FA]/)[CD] [DF][CA] |"G" .[B,G].[B,G]"_Fine".[B,G] z :| !
|: [K:D]"_II""D" [FA][df] [ce][Bd] |"A7" [Ac]([Ac]/[ce]/) [EG]2 |
"A7" [GA][Bg] [Af][Ge] |"D" [Fd]([Fd]/[Af]/) [FA]2 |
"D" [FA][df] [ce][Bd] |"A7" [Ac]([Ac]/[ce]/) [EG]2 | 
"A7" [GA][Bg] [Af][Ge] |"D" .[Fd].[Fd]"_I, dann Trio".[Fd] z :| !
|: "_Trio" "C" [EG]([Gc]/[ce]/) .[eg].[eg] |"C" [CE]([EG]/[Gc]/) .[ce].[ce] | 
"G7" [Bd][FG] [FA][GB] |"C" [Ec][ce] [eg]2 |
"C" [EG]([Gc]/[ce]/) .[eg].[eg] | "C" [CE]([EG]/[Gc]/) .[ce].[ce] |
"G7" [Bd][FG] [FA][GB] |"C" .[Ec].[Ge]"_I".[Ec] z :|
~~~~

**Comment by paulrosen** [comment]
I have reproduced this. I'll try to get a fix in the upcoming release.

**Comment by paulrosen** [comment]
Will be fixed in 5.3.0
